packages/flutter/test/widgets/draggable_scrollable_sheet_test.dart

```diff
@@ -1746,4 +1746,31 @@ void main() {
       areCreateAndDispose,
     );
   });
+
+  testWidgets('DraggableScrollableSheet respects shouldCloseOnMinExtent',
+      (WidgetTester tester) async {
```

Comment:
tiny whitespace suggestion:

```suggestion
  testWidgets('DraggableScrollableSheet respects shouldCloseOnMinExtent', (WidgetTester tester) async {
```